### PR TITLE
Code change for Index page and Category page

### PR DESCRIPTION
**Pull request by nickhsine**

### CODE REVIEW

**Comment by hcchien** [comment]
👍

**Comment by nickhsine** [comment]
@hcchien update Category page as well. Please have a see.

File: src/containers/Category.js [hunk 1]
```diff
@@ -1,67 +1,97 @@
...
 import { connect } from 'react-redux'
-import { denormalizeArticles } from '../utils/index'
-import { fetchCategorizedArticlesIfNeeded } from '../actions/group-articles'
+import { denormalizeArticles, getCatId } from '../utils/index'
+import { fetchArticlesByCatIdIfNeeded } from '../actions/articles'
 import _ from 'lodash'
-import catToTag from '../conf/category-tag-mapping-table'
 import Footer from '../components/Footer'
+import React, { Component } from 'react'
 import Tags from '../components/Tags'
```
Comment (hcchien) [inline src/containers/Category.js]: mark a TODO to rename the component 'Tags'.

**Comment by hcchien** [comment]
LGTM

**Pull request merged**